lib/mapObjects/CRewardableConstructor.cpp

```diff
@@ -63,17 +63,26 @@ TRewardLimitersList CRandomRewardObjectInfo::configureSublimiters(CRewardableObj
 
 void CRandomRewardObjectInfo::configureLimiter(CRewardableObject * object, CRandomGenerator & rng, CRewardLimiter & limiter, const JsonNode & source) const
 {
+	std::vector<SpellID> spells;
+	for (size_t i=0; i<6; i++)
+		IObjectInterface::cb->getAllowedSpells(spells, static_cast<ui16>(i));
+
+
 	limiter.dayOfWeek = JsonRandom::loadValue(source["dayOfWeek"], rng);
 	limiter.daysPassed = JsonRandom::loadValue(source["daysPassed"], rng);
-	limiter.minLevel = JsonRandom::loadValue(source["minLevel"], rng);
+	limiter.heroExperience = JsonRandom::loadValue(source["heroExperience"], rng);
```

Comment:
Do we need a limiter for hero experience? Isn't the level enough?

...besides I don't think many people can tell from of their head ie. what level do you get for 100000 experience or vice versa.

Reply:
Don't expect it to be used actively, mostly added for consistency with rewards.
Yes, checking for level should be preferred way.